groups/wg-device-management/groups.yaml

```diff
@@ -34,3 +34,28 @@ groups:
       - patrick.ohly@intel.com
     members:
       - wg-device-management-leads@kubernetes.io
+
+  #
+  # k8s-staging write access for WG-owned subprojects
+  #
+  # Each group here represents privileged access to a staging project,
+  # allowing the members to directly write to GCS and GCR within the
+  # project, as well as trigger Cloud Build within the project. Ideally
+  # this level access is used solely for troubleshooting purposes.
+  #
+  # Membership should correspond roughly to subproject owners for the set of
+  # subproject artifacts being stored in a given staging project
+  #
+
+  - email-id: k8s-infra-staging-dra-example-driver@kubernetes.io
+    name: k8s-infra-staging-dra-example-driver
+    description: |-
+      ACL for pushing dra-example-driver artifacts
+    settings:
+      ReconcileMembers: "true"
+    members: # https://github.com/kubernetes-sigs/dra-example-driver/blob/main/OWNERS
+      - alexey.fomenko@intel.com
+      - eduard.bartosh@intel.com
+      - elezar@nvidia.com
+      - klueska@gmail.com
+      - patrick.ohly@intel.com
```

Comment:
FYI @bart0sh @elezar @klueska @pohly @byako

Reply:
/lgtm

Reply:
@nojnhuh: what about adding yourself?

I certainly hope that you will continue to help with maintaining this! :wink:

Reply:
Yep I can definitely add myself! Should I include myself in the OWNERS file added here too?

Reply:
I added myself here but not in the OWNERS file, which I understand means something different.

Reply:
I think it's fine to also have you there. If I remember correctly, we trigger image promotion by creating and updating files there. It seems reasonable to grant you the right to approve that when you also have permission to do that manually.

Reply:
Sounds good, done!